CHANGELOG.md

```diff
@@ -4,6 +4,55 @@ All notable changes to Zebra are documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.0.0-rc.0](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-rc.0) - 2022-10-12
+
+This is the first Zebra release candidate. Zebra's consensus rules, node sync, and `lightwalletd` RPCs are ready for user testing and experimental use. Zebra has not been audited yet.
+
+This release also makes significant performance improvements to RPCs, and temporarily removes support for Rust 1.64.
+
+### Breaking Changes
+
+This release has the following breaking changes:
+- Rust 1.64 is unsupported due to a performance regression when downloading the Zcash parameters.
+  Zebra currently builds with Rust 1.63 ([#5251](https://github.com/ZcashFoundation/zebra/pull/5251)).
+- Use correct TOML syntax in Docker zebrad.toml ([#5320](https://github.com/ZcashFoundation/zebra/pull/5320))
+
+### Major RPC Performance Improvements
+
+This release improves RPC performance:
+- Initial `lightwalletd` sync is about twice as fast ([#5307](https://github.com/ZcashFoundation/zebra/pull/5307))
+- RPCs can run while a block is being committed to the state, previously they could be delayed by 5-15 seconds ([#5134](https://github.com/ZcashFoundation/zebra/pull/5134), [#5257](https://github.com/ZcashFoundation/zebra/pull/5257))
+
+### Security
+
+- Make default command work in docker images, disable optional listener ports ([#5313](https://github.com/ZcashFoundation/zebra/pull/5313))
+- Update yanked versions of cpufeatures (orchard/aes), ed25519 (tor), and quick-xml (flamegraph) ([#5308](https://github.com/ZcashFoundation/zebra/pull/5308))
+- Bump rand\_core from 0.6.3 to 0.6.4, fixes unsoundness bug ([#5175](https://github.com/ZcashFoundation/zebra/pull/5175))
```

Comment:
```suggestion
- Bump rand_core from 0.6.3 to 0.6.4, fixes unsoundness bug ([#5175](https://github.com/ZcashFoundation/zebra/pull/5175))
```

Reply:
The escape avoids activating italics if there are multiple underscores.

This renders ok in Markdown:
<img width="822" alt="Screen Shot 2022-10-12 at 10 53 11" src="https://user-images.githubusercontent.com/8951843/195224410-d5d76acc-80cf-469e-9ac3-bb4228c1bfef.png">